web/docs/formats/syslog.md

```diff
@@ -0,0 +1,66 @@
+# syslog
+
+Reads syslogs.
```

Comment:
I think we should stick to the singular "syslog".

Reply:
There are a few more remaining occurrences of "syslogs". You can also leave it in, but based on my quick searches the plural form is almost never used.